SSA/Projects/LLVMRiscV/PeepholeRefine.lean

```diff
@@ -22,9 +22,9 @@ our rewrites into a form accepted by the Peephole Rewriter.
 structure for LLVM `Com`s. The refinement is based on the
 dedicated refinement relation for the `PoisonOr` type, where
 a poison value can be refined by any concrete value. -/
-structure LLVMPeepholeRewriteRefine (Γ : Ctxt Ty) where
-  lhs : Com LLVMPlusRiscV Γ .pure (Ty.llvm (.bitvec 64))
-  rhs : Com LLVMPlusRiscV Γ .pure (Ty.llvm (.bitvec 64))
+structure LLVMPeepholeRewriteRefine (w : InstCombine.Width 0) (Γ : Ctxt Ty) where
```

Comment:
```suggestion
structure LLVMPeepholeRewriteRefine (w : Nat) (Γ : Ctxt Ty) where
```
Why not make this a nat, since you're using it as `toConcrete` anyway!